samples/.gitignore

```diff
@@ -0,0 +1 @@
+.generated.eml.yaml
```

Comment:
I noticed this being left behind, not sure if it should be cleaned up by the app.